Simplified/it.lproj/Localizable.strings

```diff
@@ -143,13 +143,13 @@
 "Next Chapter" = "Capitolo successivo";
 
 // NYPLReaderSettingsView
-"Special Font" = "Font speciale";
-"Sans" = "Sans";
+"Special Font" = "Font personalizzato";
+"Sans" = "Sans serif";
```

Comment:
The key should be `Sans Serif` like it's in the En localizable strings

Reply:
great catch, fixed